gstreamer/src/message.rs

```diff
@@ -1230,7 +1214,7 @@ macro_rules! message_builder_generic_impl {
 pub struct EosBuilder<'a> {
     src: Option<Object>,
     seqnum: Option<Seqnum>,
-    other_fields: Vec<(&'a str, &'a ToSendValue)>,
+    #[allow(unused)] other_fields: Vec<(&'a str, &'a ToSendValue)>,
```

Comment:
I think we should factorize the attributes `other_fields` in a generic struct `MessageBuilder`.

Reply:
Ok